css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,61 @@
+@import "../page-vars";
+
+.lune amp-sidebar {
+  width: 80vw;
+}
+
+.lune amp-sidebar .ampstart-navbar-trigger  {
+  padding-top: .625rem; /* 10px */
+}
+
+/* On large device the width should be set to 50vw */
+@media (--breakpoint-lg) {
+  .lune amp-sidebar {
+    width: 50vw;
+  }
+}
+
+/* The design has a transparent mask, this selector passes the validator */
+.lune div[class*='sidebar-mask'] {
+  opacity: 0;
+}
+
+.lune .ampstart-sidebar-nav {
+  position: absolute;
+  top: 50%;
+  left: 50%;
```

Comment:
This is for centering the content in the sidebar, but we can look into positioning it with flexbox instead.